### DISCUSSION TITLE
`odo service list` and Service Binding

### DISCUSSION THREAD

**Discussion by dharmit** [redhat-developer/odo#4844] [General]
Currently, a Service Binding created by `odo link` shows up in `odo service list` output. There was some discussion on some call about preventing bindings from showing up in that output. Just like `odo unlink`, `odo service delete` removes a Service Binding. But this makes things a bit ambiguous, as we have two ways of removing a binding/link.

I think we should prevent Service Bindings from showing up in `odo service list`. I think this way because:
1. seeing it in service list output could confuse a user because they never created it using `odo service create`. If this prompts her to delete it using `odo service delete`, it would lead to unlinking, which might be even more frustrating because she doesn't know why the link isn't working any more.
2. a user probably doesn't care seeing it in service list output; she only cares about link working as they would expect it.

**Comment by dharmit** [comment]
@kadel @feloy thoughts?

**Comment by kadel** [comment]
> I think we should prevent Service Bindings from showing up in `odo service list`

+1

**Reply by feloy** [reply]
Agree, the fact that a link is itself a service is confusing, and should be hidden to the end user

**Reply by kadel** [reply]
created https://github.com/openshift/odo/issues/4861
